lib/findPort.js

```diff
@@ -33,21 +33,42 @@ module.exports = function(host, start, endOrCallback, callback) {
 	if (typeof ports != 'undefined' && ports.length > 0) {
 		async.filter(ports, probe, callback);
 	} else {
-		(new Promise(function(resolve, reject) {
+		(function() {
```

Comment:
Is it possible to avoid this huge IIFE inside the `if` clause here?

Reply:
I think that was addressed in the the fe7584f5cbfcec2cd299f6c5e2a7a04133d46f56 commit?

Reply:
Yup, donno how I missed that! :O